src/core/sagas/utils.js

```diff
@@ -1,9 +1,7 @@
-/* @flow */
```

Comment:
I removed it because I find that syntax really head-scratching and it really polluted an otherwise quite straightforward file. I honestly have little idea looking at that function signature what that code does anymore, and I can't imagine a relative ES6 newcomer groking it well.